mmpose/datasets/pipelines/bottom_up_transform.py

```diff
@@ -733,41 +976,13 @@ def __call__(self, results):
             input_size = np.array([input_size, input_size], dtype=np.int)
         img = results['img']
 
-        h, w, _ = img.shape
-
-        # calculate the size for min_scale
-        min_input_w = _ceil_to_multiples_of(self.min_scale * input_size[0], 64)
```

Comment:
please double check this part.

Reply:
Thanks, the option `use_udp` is lost. After fixing this problem, I compare the deleted codes with the function `_get_multi_scale_size` and find that they match well. On 10000 random inputs, two versions of `BottomUpGetImgSize` generate the same outputs